package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opennms",
-  "version": "1.5.1-SNAPSHOT",
+  "version": "2.0.0-SNAPSHOT",
```

Comment:
Is bumping the major necessary? I understand that we made some changes to the base DAOs, but are we expecting users of the library to actually extend/implement these interfaces?

Reply:
Most likely not, but it does technically change the "public" API of DAOs.  Also the logging changes I make in the other PR here changes the API bundle too, so between the two we're going to 2.0.0 either way.

Reply:
👍